Header header
bool valid
float64 latitude
float64 longitude
float64 altitude
float64 ground_speed
int32 satellites
string direction